telemetry/definitions/commonDefinitions.json

```
@@ -1309,6 +1348,11 @@
             "type": "int",
             "description": "Time from opening Composer to loading the file"
         },
+        {
+            "name": "localBuildSystemVersion",
```

everything is assumed to be on the user's machine. use "remote" to call out when it's not. usually "local" is not necessary to call out.
```suggestion
            "name": "buildSystemVersion",
```